src/xb-main.c

```diff
@@ -90,8 +90,7 @@ server_send_response (SoupMessage *message,
 
 static char *
 get_real_path (const char *path,
```

Comment:
i would have also recommended you rename this to `resolve_flatpak_path` or something similar, since it wasn't clear this was about flatpak...